forge-game/src/main/java/forge/game/ability/effects/ImmediateTriggerEffect.java

```diff
@@ -68,7 +68,9 @@ public void resolve(SpellAbility sa) {
             if (overridingSA instanceof AbilitySub) {
                 ((AbilitySub)overridingSA).setParent(null);
             }
-
+            if (sa.hasParam("CopyTriggeringObjects")) {
```

Comment:
I made that obsolete by introducing "Spawner>..." syntax
But in this case it's only for itself anyway, so nothing should prevent you from using normal Count$